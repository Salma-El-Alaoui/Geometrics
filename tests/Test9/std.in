DELETE c1 pl1
C c1 1 1 1
PL pl1 1 1 1 1 1 1 1 1
LIST
DELETE c1 pl1 pl2
LIST
R r1 1 1 1 1
L l1 1 1 1 1
LIST
DELETE r1 l1 123
LIST
DELETE c1 pl1
LIST
EXIT
