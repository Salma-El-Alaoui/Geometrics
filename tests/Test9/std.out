ERR
#L'element geometrique nomme c1 n'existe pas.
OK
#Element ajoute: c1
OK
#Element ajoute: pl1
C c1 1 1 1
PL pl1 1 1 1 1 1 1 1 1
ERR
#L'element geometrique nomme pl2 n'existe pas.
C c1 1 1 1
PL pl1 1 1 1 1 1 1 1 1
OK
#Element ajoute: r1
OK
#Element ajoute: l1
C c1 1 1 1
L l1 1 1 1 1
PL pl1 1 1 1 1 1 1 1 1
R r1 1 1 1 1
ERR
#Parametres invalides
C c1 1 1 1
L l1 1 1 1 1
PL pl1 1 1 1 1 1 1 1 1
R r1 1 1 1 1
#Element c1 supprime
#Element pl1 supprime
OK
L l1 1 1 1 1
R r1 1 1 1 1
